### PR TITLE
chore: update config and a default wallet connect key

### PR DESCRIPTION
**Pull request by molecula451**
Refer #815 

This  fixes the "annoying" invalid key error anytime the local env or the whole dapp env does not read the WalletConnect Key, this  key is free and it can be generate many times, so no  a security concern

Screenshot from 2024-01-26 12-38-17

### CODE REVIEW

**Comment by gitcoindev** [comment]
Hi @molecula451 I see the build is broken, could you please try to fix this? I will do the review today.

```
Failed to compile.
./components/lib/hooks/use-web-3.tsx:12:23
Type error: Cannot find module 'ubq-connect.config' or its corresponding type declarations.
```

**Comment by molecula451** [comment]
> Hi @molecula451 I see the build is broken, could you please try to fix this? I will do the review today.

fixed at  https://github.com/ubiquity/ubiquity-dollar/pull/888/commits/d02ce8a0e49186339cd3e791cca35d37200e6ccf missed the latest change and push

File: packages/dapp/tsconfig.json [hunk 1]
```diff
@@ -26,6 +26,6 @@
       "@/types/contracts": ["types/contracts"]
     }
   },
-  "include": ["global.d.ts", "next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["global.d.ts", "next-env.d.ts", "**/*.ts", "**/*.tsx", "ubq.config.ts"],
```
Comment (gitcoindev) [inline packages/dapp/tsconfig.json]: The include should be updated to `ubq-connect-config.ts`

**Review by gitcoindev — CHANGES_REQUESTED** [review]

File: packages/dapp/tsconfig.json (hunk 1 above)
Comment (molecula451) [reply packages/dapp/tsconfig.json]: pleased at https://github.com/ubiquity/ubiquity-dollar/pull/888/commits/df1e6deda304c2ed4494e030e2cfb2748a791848

**Comment by molecula451** [comment]
btw i generated the key myself so its not something that its laying around the web lol

**Review by gitcoindev — APPROVED** [review]

**Review by rndquu — APPROVED** [review]

**Pull request merged**